docs/source/ecf.rst

```diff
@@ -168,6 +168,10 @@ As we want to do our own spectral extraction, we set this variable to ``calints`
 .. note::
 	Note that other Instruments might used different suffixes!
 
+photometry
+''''''''''
+Only used for photometry analyses. Set to True if the user wants to analyze a photometric dataset.
+
```

Comment:
The reason why I added that is because of the directory-naming update (like the ap65bg90150 I mentioned above). The directories which use the ap and bg sizes are being created **before** the fits files are being read in. 
I guess, I could instead check if the ecf contains photap, skyin and skyout and then assume that the person is doing photometry... but I am not sure if that's a good solution.

Reply:
Hmm, I see your point. What you have works for now, so let's just stick with that for the time being